plugin/evm/customheader/min_delay_excess.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package customheader
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/vms/evm/acp226"
+	"github.com/ava-labs/libevm/core/types"
+
+	"github.com/ava-labs/coreth/params/extras"
+	"github.com/ava-labs/coreth/plugin/evm/customtypes"
+)
+
+var (
+	errRemoteMinDelayExcessSet = errors.New("remote min delay excess should be nil")
+	errRemoteMinDelayExcessNil = errors.New("remote min delay excess should not be nil")
+	errIncorrectMinDelayExcess = errors.New("incorrect min delay excess")
+	errParentMinDelayExcessNil = errors.New("parent min delay excess should not be nil")
+)
+
+// MinDelay
+//
+// If the `desiredMinDelayExcess` is nil, the parent's delay excess is used.
```

Comment:
nit: improvement on the godoc of this function:
```suggestion
// MinDelayExcess computes the min delay excess to include in the header.
// If the `desiredMinDelayExcess` is nil, the parent's delay excess (or the
// initial delay excess at activation) is used.
```